packages/fast-layouts-react/src/canvas/canvas.tsx

```diff
@@ -22,7 +22,7 @@ class Canvas extends Foundation<CanvasProps, undefined> {
      * Default props for the Canvas component
      */
     public static defaultProps: ICanvasHandledProps = {
-        minWidth: 300
+        minWidth: 300,
```

Comment:
`toPx()`?

Reply:
These are set as props and not as JSS styles, so I don't think we would want to require JSS utilities to accommodate the default props.